<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Overlay</class>
 <widget class="QMainWindow" name="Overlay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="inputNumber">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="outputNumber">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="enableBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>98</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>enable</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="inputOver">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>259</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
